pkg/slack/client.go

```diff
@@ -0,0 +1,49 @@
+package slack
```

Comment:
I presume similarly to `resend` this will be moved to a good old top-level package.